examples/guestbook/README.md

```diff
@@ -37,44 +37,99 @@ Documentation for other releases can be found at
 
-### Step Zero: Prerequisites
+### Prerequisites
+
+This example requires a running Kubernetes cluster. See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. And follow the [Prerequisites](../../docs/user-guide/prereqs.md) to make sure your `kubectl` is ok. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
+
+### Quick Start
+
+This section shows a simplest way to get the example work. If you want to know the details, you should skip this and read [the rest of the example](#step-one-start-up-the-redis-master).
+
+Start the guestbook with one command:
+
+```console
+$ kubectl create -f examples/guestbook/all-in-one/guestbook-all-in-one.yaml
+service "redis-master" created
+replicationcontroller "redis-master" created
+service "redis-slave" created
+replicationcontroller "redis-slave" created
+service "frontend" created
+replicationcontroller "frontend" created
+```
+
+You also can start the guestbook by running:
+
+```console
+$ kubectl create -f examples/guestbook/
+```
+
+Then, list all your services:
+
+```console
+$ kubectl get services
+NAME              CLUSTER_IP       EXTERNAL_IP       PORT(S)       SELECTOR                            AGE
+frontend          10.0.93.211      <none>            80/TCP        app=guestbook,tier=frontend         1h
+redis-master      10.0.136.3       <none>            6379/TCP      app=redis,role=master,tier=backend  1h
+redis-slave       10.0.21.92       <none>            6379/TCP      app=redis,role=slave,tier=backend   1h
+```
 
-This example requires a running Kubernetes cluster.  See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
+Now you can access the guestbook on each node with frontend service's `<ClusterIP>:Port`, e.g. `10.0.93.211:80` in this guide. `<ClusterIP>` is an a cluster-internal IP. If you want to access the guestbook from outside of the cluster, add `type: NodePort` to frontend service `spec` field. Then you can access the guestbook with `<NodeIP>:NodePort` from outside of the cluster. There are other ways for you to access the guestbook. You may learn from [Accessing services running on the cluster](../../docs/user-guide/accessing-the-cluster.md#accessing-services-running-on-the-cluster).
+
+Clean up the guestbook:
+
+```console
+$ kubectl delete -f examples/guestbook/all-in-one/guestbook-all-in-one.yaml
+```
+or
+```console
+$ kubectl delete -f examples/guestbook/
+```
+
+Before continuing to the gory details, we also recommend you to read [Quick walkthrough](../../docs/user-guide/#quick-walkthrough), [Thorough walkthough](../../docs/user-guide/#thorough-walkthrough) and [Concept guide](../../docs/user-guide/#concept-guide).
```

Comment:
Suggest moving this paragraph to the beginning of the "Step One" section, so that when people skip "Quick Start" they won't skip this too.